xstate-chatbot/nodejs/src/machine/vitals.js

```diff
@@ -348,6 +348,7 @@ const vitalsFlow = {
                 context.slots.person.com_status = context.intention.com_status;
                 context.slots.person.fateh_kit_delivered = context.intention.fateh_kit_delivered;
                 context.slots.person.data_source_type = context.intention.data_source_type;
+                context.slots.person.srfC_id=context.intention.srf_Id;
```

Comment:
Does this work properly? 

`context.intention` is a string at most of the places.

Reply:
i have tested with cova and data is getting stored in cova reports after complete flow

Reply:
Still, I suggest don't store anything in context.intention. It is only for storing intention which is a string. To implement this change you would have to make changes where you are storing values to context.intention.

Reply:
ok

Reply:
code changes committed